error: ‘notmuch_config_path’ redeclared as different kind of symbol

I am getting the following error when trying to compile:

```shell
notmuch-addrlookup.c:21:15: error: ‘notmuch_config_path’ redeclared as different kind of symbol
   21 | static gchar* notmuch_config_path = NULL;
```

I am on Fedora 34.

Full log:

```shell
cc -Wall -O2 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/sysprof-4 -pthread  -std=c99   -c -o notmuch-addrlookup.o notmuch-addrlookup.c
notmuch-addrlookup.c:21:15: error: ‘notmuch_config_path’ redeclared as different kind of symbol
   21 | static gchar* notmuch_config_path = NULL;
      |               ^~~~~~~~~~~~~~~~~~~
In file included from notmuch-addrlookup.c:9:
/usr/include/notmuch.h:2756:1: note: previous declaration of ‘notmuch_config_path’ with type ‘const char *(notmuch_database_t *)’ {aka ‘const char *(struct _notmuch_database *)’}
 2756 | notmuch_config_path (notmuch_database_t *notmuch);
      | ^~~~~~~~~~~~~~~~~~~
notmuch-addrlookup.c: In function ‘create_queries’:
notmuch-addrlookup.c:176:3: warning: ‘notmuch_query_count_messages_st’ is deprecated: function deprecated as of libnotmuch 5.0 [-Wdeprecated-declarations]
  176 |   if (notmuch_query_count_messages_st (queries[0], &tmp) == NOTMUCH_STATUS_SUCCESS)
      |   ^~
In file included from notmuch-addrlookup.c:9:
/usr/include/notmuch.h:1279:1: note: declared here
 1279 | notmuch_query_count_messages_st (notmuch_query_t *query, unsigned int *count);
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
notmuch-addrlookup.c:178:3: warning: ‘notmuch_query_count_messages_st’ is deprecated: function deprecated as of libnotmuch 5.0 [-Wdeprecated-declarations]
  178 |   if (notmuch_query_count_messages_st (queries[1], &tmp) == NOTMUCH_STATUS_SUCCESS)
      |   ^~
In file included from notmuch-addrlookup.c:9:
/usr/include/notmuch.h:1279:1: note: declared here
 1279 | notmuch_query_count_messages_st (notmuch_query_t *query, unsigned int *count);
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
notmuch-addrlookup.c: In function ‘run_queries’:
notmuch-addrlookup.c:247:7: warning: ‘notmuch_query_search_messages_st’ is deprecated: function deprecated as of libnotmuch 5.0 [-Wdeprecated-declarations]
  247 |       if (notmuch_query_search_messages_st (queries[i], &messages) != NOTMUCH_STATUS_SUCCESS)
      |       ^~
In file included from notmuch-addrlookup.c:9:
/usr/include/notmuch.h:1183:1: note: declared here
 1183 | notmuch_query_search_messages_st (notmuch_query_t *query,
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/notmuch-addrlookup.c
+++ b/notmuch-addrlookup.c
@@ -18,12 +18,12 @@ static gchar* notmuch_database_path = NULL;
 static gchar* notmuch_user_email = NULL;
 static gchar** search_terms = NULL;
 static gboolean mutt_output = FALSE;
-static gchar* notmuch_config_path = NULL;
+static gchar* notmuch_config_path_lookupvar = NULL;
 
 static const GOptionEntry option_entries[] = {
   { "mutt", 'm', 0, G_OPTION_ARG_NONE, &mutt_output,
     "Format output for Mutt", NULL },
-  { "config", 'c', 0, G_OPTION_ARG_STRING, &notmuch_config_path,
+  { "config", 'c', 0, G_OPTION_ARG_STRING, &notmuch_config_path_lookupvar,
     "Path to config file .notmuch-config", NULL },
   { G_OPTION_REMAINING, 0, 0, G_OPTION_ARG_STRING_ARRAY, &search_terms,
     "Search terms", NULL },
@@ -97,10 +97,10 @@ load_notmuch_settings (void)
 
   if (g_getenv ("NOTMUCH_CONFIG"))
     config_path = g_strdup (g_getenv ("NOTMUCH_CONFIG"));
-  else if(!notmuch_config_path)
+  else if(!notmuch_config_path_lookupvar)
     config_path = g_strdup_printf ("%s/.notmuch-config", g_get_home_dir ());
   else
-    config_path = notmuch_config_path;
+    config_path = notmuch_config_path_lookupvar;
 
 
   if (!g_key_file_load_from_file (key_file,

